tasks.mak

```diff
@@ -128,7 +128,7 @@ dist/opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit:
 	bash ./builds/fluent_bit.sh $(PWD)/dist
 
 dist/opt/google-cloud-ops-agent/subagents/opentelemetry-collector/otelopscol:
-	SKIP_OTEL_JAVA=${SKIP_JAVA} bash ./builds/otel.sh $(PWD)/dist
+	SKIP_OTEL_JAVA=${SKIP_JAVA} GO_BIN=$(shell which go) bash ./builds/otel.sh $(PWD)/dist
```

Comment:
Could this potentially pick an "incorrect" Go version from the system? For example, if builds machines have an existing go installation, but we also install a specific version for builds.

Reply:
This target is only used for local development and won't affect our CI builders. They will still invoke the script without providing a `GO_BIN` which will cause it to default to `/usr/local/go/bin/go` which is where Go is installed in all build containers.